src/OscapScannerRemoteSsh.cpp

```diff
@@ -121,8 +121,9 @@ QStringList OscapScannerRemoteSsh::getCommandLineArgs() const
     }
     else
     {
+        QString userTailoringFile = mSession->getUserTailoringFilePath();
         args += buildEvaluationArgs(mSession->getOpenedFilePath(),
-            mSession->hasTailoring() ? mSession->getTailoringFilePath() : QString(),
+            mSession->hasTailoring() ?  ( userTailoringFile.isEmpty() ? mSession->getTailoringFilePath() : userTailoringFile ): QString(),
```

Comment:
IMHO, you could create ScanningSession method to wrap these double ternary operators